plone/app/contentlisting/tests/test_integration_unit.py

```diff
@@ -262,6 +266,14 @@ def test_item_get_none(self):
                 "result in an AttributeError."
             )
 
+    def test_item_getattr_acquisition(self):
+        self.portal.get("test-folder").setTitle("Test folder")
+        self.item._realobject.__class__.Title = patched_Title
```

Comment:
@gbastien This is being left in place after this test completes and breaking a lot of other tests. I'd suggest using https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch as a context manager to patch the Title method in a way that will be automatically removed whether or not there is an error in the test.

Reply:
Hi @davisagli 

thank you for follow up.  Actually I could avoid the monkeypatch by using the absolute_url method that needs the acquisition to work as discovered by test test_special_getattr_from_object that is now succeeding in calling absolute_url.

I will adapt code and come back to you.

Thank you,

Gauthier